packages/package-manager/src/NodePackageManagers.ts

```diff
@@ -96,19 +96,28 @@ export class NpmPackageManager implements PackageManager {
     await this._runAsync(['install', '--global', ...names]);
   }
 
-  async addAsync(...names: string[]) {
+  async addWithParametersAsync(names: string[], parameters: string[] = []) {
     if (!names.length) return this.installAsync();
```

Comment:
In this case, wouldn't you want to still use parameters?

Reply:
Yes, you would. Good catch.